espnet/nets/pytorch_backend/e2e_asr_transformer.py

```diff
@@ -110,6 +110,15 @@ def __init__(self, idim, odim, args, ignore_id=-1):
         else:
             self.ctc = None
 
+        if 'report_cer' in vars(args) or args.mtlalpha > 0.0:
```

Comment:
why not `if args.report_cer`?

Reply:
What happens if `report_cer == False && report_wer == True`?

Reply:
I think that this `if` detects that `report_cer` has been declared in `args` regardless its value ( in case of `None`)

Reply:
I think this is better because `report_cer` and `report_wer` should not be None as defined in `asr_train.py`.
```python
if args.report_cer or args.report_wer:
    self.error_calculator(...)
```